.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
     hooks:
       - id: terraform-docs-go
         args: ["markdown", "table", "--output-file", "README.md", "./"]
+        files: ^(\.github\/workflows\/.*|.*\.hcl|.*\.env|.*\.tf)$
```

Comment:
@rsalmond, is this the right set of paths to run TF checks on:
- '**/*.tf'
- '**/*.env'
- '**/*.hcl'
- '.github/workflows/**'